astropy/coordinates/earth_orientation.py

```diff
@@ -97,34 +97,24 @@ def obliquity(jd, algorithm=2006):
 
     References
     ----------
-    * Hilton, J. et al., 2006, Celest.Mech.Dyn.Astron. 94, 351. 2000
-    * USNO Circular 179
+    * Hilton, J. et al., 2006, Celest.Mech.Dyn.Astron. 94, 351
+    * Capitaine, N., et al., 2003, Astron.Astrophys. 400, 1145-1154
     * Explanatory Supplement to the Astronomical Almanac: P. Kenneth
       Seidelmann (ed), University Science Books (1992).
     """
-    T = (jd - jd2000) / 36525.0
-
     if algorithm == 2006:
-        p = (-0.0000000434, -0.000000576, 0.00200340, -0.0001831, -46.836769, 84381.406)
-        corr = 0
+        return np.rad2deg(erfa.obl06(jd, 0))
     elif algorithm == 2000:
-        p = (0.001813, -0.00059, -46.8150, 84381.448)
-        corr = -0.02524 * T
+        return np.rad2deg(erfa.pn00(jd, 0, 0, 0)[0])
```

Comment:
This seems a bit lazy to me. I would suggest calling `erfa.pr00()` to get the obliquity correction to `erfa.obl80()` as does `erfa.pn00()` without all the extraneous stuff that `erfa.pn00()` does also.

Reply:
Heh, I wouldn't call it "lazy", more that I wanted to stick to call as few ERFA functions as possible.  But, you're right: especially with the clarity from the docstring description, it's better to simply call two ERFA functions and add the output.